# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-sparcv9.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-118.0.2/build/moz.configure/init.configure.vanilla	2023-10-14 18:25:26.242187047 +0200
+++ firefox-118.0.2/build/moz.configure/init.configure	2023-10-14 18:26:00.505187945 +0200
@@ -563,7 +563,7 @@
     elif cpu.startswith("hppa") or cpu == "parisc":
         canonical_cpu = "hppa"
         endianness = "big"
-    elif cpu.startswith("sparc64") or cpu.startswith("sparcv9"):
+    elif cpu.startswith("sparc64"):
         canonical_cpu = "sparc64"
         endianness = "big"
     elif cpu.startswith("sparc") or cpu == "sun4u":
